metrics/windows/filesystem.go

```diff
@@ -13,33 +13,35 @@ import (
 
 // FilesystemGenerator XXX
 type FilesystemGenerator struct {
+	IgnoreRegexp *regexp.Regexp
 }
 
 // NewFilesystemGenerator XXX
-func NewFilesystemGenerator() (*FilesystemGenerator, error) {
-	return &FilesystemGenerator{}, nil
+func NewFilesystemGenerator(ignoreReg *regexp.Regexp) (*FilesystemGenerator, error) {
```

Comment:
You need to pass `ignoreReg` on `command/command_windows.go`, like *nix oses (ref: https://github.com/mackerelio/mackerel-agent/blob/master/command/command_linux.go#L32 )